src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+
+import { useEffect, useState } from 'react';
+import { Todo as TypeTodo } from '../../types/Todo';
+
+type Props = {
+  todo: TypeTodo;
+  handleDelete: (id: number[]) => void;
+  toDelete: boolean;
+};
+
+export const Todo: React.FC<Props> = ({ todo, handleDelete, toDelete }) => {
+  const [loading, setLoading] = useState(false);
+
+  const checkTempTodo = (id: number) => {
+    const TEMP_TODO_ID = 0;
+
+    return id === TEMP_TODO_ID;
+  };
+
+  useEffect(() => {
+    if (checkTempTodo(todo.id)) {
+      setLoading(true);
+    }
+  }, [todo.id]);
```

Comment:
I think that this might work alright without state. If todo is changed then this component will rerender.


```suggestion
 const checkTempTodo = (id: number) => {
    const TEMP_TODO_ID = 0;

    return id === TEMP_TODO_ID;
  };

  const isLoading = checkTempTodo(todo.id)
```